templates/web/camden/auth/general.html

```diff
@@ -0,0 +1,88 @@
+[% INCLUDE 'header.html', bodyclass='authpage', title = loc('Sign in or create an account') %]
+
+<h1>
+    [% loc('Sign in') %]
+    <small>
+        [% tprintf(loc('or <a href="%s">create an account</a>'), '/auth/create') %]
+    </small>
+</h1>
+
+[% TRY %][% INCLUDE 'auth/_general_top.html' %][% CATCH file %][% END %]
+
+[% IF oauth_need_email %]
+    <p class="form-error">[% loc('We need your email address, please give it below.') %]</p>
+[% END %]
+[% IF oauth_failure %]
+    <p class="form-error">[% loc('Sorry, we could not log you in. Please fill in the form below.') %]</p>
+[% END %]
+
+<form action="/auth" method="post" name="general_auth" class="validate">
+    <fieldset>
+
+        <input type="hidden" name="r" value="[% c.req.params.r | html %]">
+
+        [% loc_username_error = INCLUDE 'auth/_username_error.html' default='email' %]
+
+[% IF c.cobrand.sms_authentication %]
+    [% SET username_label = loc('Your email or mobile') %]
+[% ELSE %]
+    [% SET username_label = loc('Your email') %]
+[% END %]
+
+        <label class="n" for="username">[% username_label %]</label>
+      [% IF loc_username_error %]
+        <div class="form-error">[% loc_username_error %]</div>
+      [% END %]
+        <input type="text" class="form-control required" id="username" name="username" value="[% username | html %]" autocomplete="username"
+        [%~ IF c.cobrand.moniker != 'borsetshire' %] autofocus[% END %]>
```

Comment:
This will always be true as we're in a Camden template :)

Reply:
Oops, ta! Have amended on the `camden-reviewed` branch.